README.md

```diff
@@ -36,14 +36,56 @@ board.on("ready", function() {
   }.bind(this), 500);
 });
 ```
+Also take a look at the demo site in the example folder.
 
-Take a look at the sketches folder to see how Firmata works together with SoftModem.
+### Sketches
 
+Three example sketches are provided:
 
-## Building
+- SimpleDigitalFirmata
+- ConfigurableFirmata
+- StandardFirmataWebJack
+
+__SimpleDigitalFirmata__ and __ConfigurableFirmata__ are capable of setting and reading digital pins. The first one is implemented with the standard Firmata library, the latter with ConfigureableFirmata.
+
+__StandardFirmataWebJack__ is a modification of the standard Firmata firmware and thus has all its capabilities. Additionally a custom SysEx command is implemented, that reports exactly one analog reading.
+
+All sketches are configured with a 10 millisecond delay in the processing loop. This is due to crosstalk of requests messages. An immediate reply will make it difficult for WebJack to distinguish the crosstalk from the actual reply. 
```

Comment:
Perhaps a link to a definition of 'crosstalk' would be helpful here. I misunderstood this term when I first saw it. Thanks!